setup.py

```diff
@@ -319,6 +364,20 @@ def _find_msbuild_tool(self, tool="msbuild.exe", use_windows_sdk=False):
 
         raise RuntimeError("{0} could not be found".format(tool))
 
+    def _find_msbuild_tool_15(self):
+        """Return full path to one of the Microsoft build tools"""
+        try:
+            basePathes = subprocess.check_output(
+                ["vswhere", "-latest",
```

Comment:
@dmitriyse can you change this to `tools\vswhere\vswhere.exe` like I did in my pull request #540?

Reply:
I added tools\vswhere\vswhere.exe file into the repository instead of chocolatey installation.
See PR #546